### PR TITLE
Bugfix 3.5: Add object counts to arangoimport latency

### PR DESCRIPTION
**Pull request by matthewvon**
This is a back port of PR 9856 (3.4).

Existing latency report stated number of requests posted to db server. But there was not tracking of how many objects / records were contained in each request. This PR utilizes the unused statistic location and populates it with total count of objects within the requests.

### CODE REVIEW

**Comment by matthewvon** [comment]
Jenkins running: http://172.16.10.101:8080/view/PR/job/arangodb-matrix-pr/6101/

File: CHANGELOG [hunk 1]
```diff
@@ -1,10 +1,12 @@
 v3.5.1 (XXXX-XX-XX)
 -------------------
+* Add count of objects to latency reporting in arangoimport
```
Comment (jsteemann) [inline CHANGELOG]:
```suggestion
* Add count of objects to latency reporting in arangoimport.
```

File: arangosh/Import/ImportHelper.cpp [hunk 2]
```diff
@@ -294,6 +294,7 @@ bool ImportHelper::importDelimited(std::string const& collectionName,
   _rowsRead = 0;
 
   char buffer[32768];
+  // int64_t totalRead = 0;
```
Comment (jsteemann) [inline arangosh/Import/ImportHelper.cpp]:
```suggestion
```

File: arangosh/Import/ImportHelper.cpp [hunk 3]
```diff
@@ -312,7 +313,7 @@ bool ImportHelper::importDelimited(std::string const& collectionName,
       break;
     }
 
-    totalRead += static_cast<int64_t>(n);
+    // totalRead += static_cast<int64_t>(n);
```
Comment (jsteemann) [inline arangosh/Import/ImportHelper.cpp]:
```suggestion
```

File: arangosh/Import/ImportHelper.cpp [hunk 4]
```diff
@@ -377,10 +378,12 @@ bool ImportHelper::importJson(std::string const& collectionName,
   }
 
   // progress display control variables
-  int64_t totalRead = 0;
+  // int64_t totalRead = 0;
```
Comment (jsteemann) [inline arangosh/Import/ImportHelper.cpp]:
```suggestion
```

File: arangosh/Import/ImportHelper.cpp [hunk 5]
```diff
@@ -419,7 +422,7 @@ bool ImportHelper::importJson(std::string const& collectionName,
       checkedFront = true;
     }
 
-    totalRead += static_cast<int64_t>(n);
+    // totalRead += static_cast<int64_t>(n);
```
Comment (jsteemann) [inline arangosh/Import/ImportHelper.cpp]:
```suggestion
```

File: arangosh/Import/QuickHistogram.h [hunk 6]
```diff
@@ -46,12 +49,14 @@ class QuickHistogram : public arangodb::Thread {
...
   QuickHistogram& operator=(QuickHistogram const&) = delete;
 
- public:
-  QuickHistogram()
-      : Thread("QuickHistogram"),
-        _writingLatencies(nullptr),
-        _readingLatencies(nullptr),
-        _threadRunning(false) {}
+public:
+QuickHistogram()
```
Comment (jsteemann) [inline arangosh/Import/QuickHistogram.h]: wrong indentation

**Comment by matthewvon** [comment]
@jsteemann I believe all code review comments are addressed in latest commit.

**Pull request merged**